tests/istio.mk

```diff
@@ -86,7 +86,7 @@ test/minikube/auth/e2e_pilot: istioctl generate_yaml
 	kubectl create ns istio-test || true
 	go test -test.v -timeout 20m ./tests/e2e/tests/pilot -args \
 		-hub ${HUB} -tag ${TAG} \
-		--skip-cleanup --mixer=true --auth=enable \
```

Comment:
ahem, how about no ?

Reply:
please don't remove this line. We would spend 2x testing time - testing with and without auth. Also, why do we have to change test makefile as part of this PR?